ci/build.py

```diff
@@ -27,63 +27,17 @@
 
 import argparse
 import glob
-import logging
-import os
+import pprint
 import re
 import shutil
+import signal
 import subprocess
-import sys
-import tempfile
 from itertools import chain
 from subprocess import check_call, check_output
 from typing import *
-from util import *
-import docker
-import docker.models
-import docker.errors
-import signal
-import atexit
-import pprint
-
-
-class Cleanup:
```

Comment:
So all of this is in the new docker client?

Reply:
Yup - it all moved to the "safe_docker_run.py" script